frontend/src/app/general/nav-bar-menu/nav-bar-menu.component.html

```diff
@@ -0,0 +1,79 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<mat-list>
+  <a mat-list-item (click)="navBarService.toggle()" routerLink="/projects"
+    >Projects</a
+  >
+  <mat-divider></mat-divider>
+  <a mat-list-item (click)="navBarService.toggle()" routerLink="">Sessions</a>
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    [routerLink]="['/sessions', 'overview']"
+    >Session overview</a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    href="/prometheus/graph"
+    target="_blank"
+    >Prometheus <mat-icon class="open-in-new">open_in_new</mat-icon></a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    href="/grafana/"
+    target="_blank"
+    >Grafana <mat-icon class="open-in-new">open_in_new</mat-icon></a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    href="/docs/"
+    target="_blank"
+    >Documentation <mat-icon class="open-in-new">open_in_new</mat-icon></a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    href="https://github.com/DSD-DBS/capella-collab-manager/issues"
+    target="_blank"
+    >Create issue on Github
+    <mat-icon class="open-in-new">open_in_new</mat-icon></a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    routerLink="settings"
+    >Settings</a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    routerLink="events"
+    >Events</a
+  >
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    routerLink="logout/redirect"
+    >Log out</a
+  >
```

Comment:
It's a pitty this is basically a copy of what can be found in the header component. Maybe the nav structure can be transformed to a data structure (as we did for breadcrumbs) and we can just render it here?

(in a separate PR)

Reply:
I was also not happy with it, but had no good solution in mind. A simple data structure should do it, thanks for the hint :)

Reply:
Moved to a data structure.